package.json

```diff
@@ -11,44 +11,43 @@
   "license": "Apache-2.0",
   "scripts": {
     "prepublish": "typings install && npm run prod",
-    "lint": "tslint -c tslint.json src/*.ts src/*.tsx src/**/*.ts src/**/*.tsx src/**/**/*.ts src/**/**/*.tsx && sass-lint -c .sass-lint.yml -v -q",
-    "test": "npm run lint && tsc && cp -r src/stylesheets lib && mocha --require lib/testHelper.js lib/__tests__/*.js lib/**/__tests__/*.js lib/**/**/__tests__/*.js",
+    "lint": "tslint -c tslint.json src/*.ts src/*.tsx src/components/*.ts src/components/*.tsx && sass-lint -c .sass-lint.yml -v -q",
+    "test": "npm run lint && tsc && cp -r src/stylesheets lib && mocha --require lib/__tests__/testHelper.js lib/__tests__/*.js",
     "test-file": "npm run lint && tsc && cp -r src/stylesheets lib && mocha --require lib/testHelper.js",
-    "dev": "rm -rf dist && NODE_ENV=development webpack --watch --progress --display-modules --config webpack.prod.config",
-    "prod": "rm -rf dist && NODE_ENV=production webpack -p --progress --display-modules --config webpack.prod.config"
```

Comment:
Webpack 4, by default, configures the `NODE_ENV` variable for you in their respective config files.

Reply:
What happened to `--watch`?

Reply:
We're not running a dev server so it's not needed at the moment.

Reply:
We are running a dev server; I've been running `npm link` and `npm run dev` on this when I'm working on importing the components from to the interfaces, so I can make changes to this as I go.  Is removing `--watch` going to make it so I can't do that...?

Reply:
That's not the same thing as a dev server. The dev server would actually start a server and serve you files on a port in localhost. This was not set up to do this from the beginning. You were using `npm run dev` before to build a local development build and then you went to the index file directly to see the changes. That is not the same as a dev server.

`--watch` watches the files as you work on them and rebuilds it but since we not using a dev server I don't think there's much point to it. We can revisit this later and build in an actual dev server.

Reply:
You can still work on this as you were before and run `npm run dev` whenever you update the package, but it's not really what you want.

Reply:
I temporarily added it back in so it can help with your dev workflow but that should be updated to be better.

Reply:
I think watch is still useful without a dev server. It won't automatically load changes, but you can update by reloading the page instead of having to manually run the command.

Reply:
Agreed

Reply:
👍 I'll leave it in then.